src/lightning_lite/strategies/ddp.py

```diff
@@ -142,6 +153,8 @@ def barrier(self, *args: Any, **kwargs: Any) -> None:
             torch.distributed.barrier()
 
     def broadcast(self, obj: TBroadcast, src: int = 0) -> TBroadcast:
+        if not _distributed_available():
+            return obj
```

Comment:
This was in the ddp spawn strategy. Merging the two broadcast implementation into one.